cmd/send-email/main.go

```diff
@@ -25,6 +27,18 @@ func main() {
         log.Fatal("Could not load config options.", err);
     }
 
+    if (args.Course != "") {
+        db.MustOpen();
+        defer db.MustClose();
```

Comment:
The DB should generally be opened as the first thing after checking the args,
since it is used for things like logging.
So even in the case where no course is provided, the DB is required.